node/manifest.ts

```diff
@@ -139,7 +142,11 @@ const generateManifest = ({
     )
 
     if (excludedPattern) {
-      manifestFunctionConfig[func.name].excluded_patterns.push(serializePattern(excludedPattern))
+      if (featureFlags?.edge_functions_excluded_patterns_on_route) {
+        route.excluded_patterns = [serializePattern(excludedPattern)]
```

Comment:
You're always assigning a one-element array here. Will it have multiple when added via ISC?

Reply:
yup - extracted that into https://github.com/netlify/edge-bundler/pull/369 as a follow-up, since I didn't want to complicate this PR. But I can also add that commit into this PR, probably makes reviewing easier.